spec/spec_helper.rb

```diff
@@ -16,7 +16,7 @@
 #
```

Comment:
I'm not yet sure where the problem is, but we seem to now be running the cypress features with rspec which obviously doesn't work. They are then marked as pending with "no such step" hence not failing on CI :worried: 
This was introduced in the first two commits of the branch. Currently tracking this down. Don't merge till this is fixed.

Reply:
Okay I think I see the problem, turnip seems to be hard-coded to load all feature files and is picking up the cypress feature files. Dang.

https://github.com/jnicklas/turnip/blob/00bd963dea49c5ff8f87edb1bcafcdb73f310c6a/lib/turnip/rspec.rb#L124

```ruby
  config.pattern += ',**/*.feature'
```

Reply:
Fixed by adding and configuring `rutabaga` gem instead of directly using `turnip`.